builders/build/substrate/adding-built-in-pallet.md

```diff
@@ -0,0 +1,201 @@
+
+The following code snippet shows an example of the constant definitions to be used in the configuration of the module:
+
+```rust
+parameter_types! {
+   // The amount of funds that must be reserved for an asset
+	pub const AssetDeposit: Balance = 100;
+   // The amount of funds that must be reserved when creating 
+   // a new transfer approval
+	pub const ApprovalDeposit: Balance = 1;
+   // The basic amount of funds that must be reserved when adding metadata 
+   // to your asset
+	pub const MetadataDepositBase: Balance = 10;
+   // The additional funds that must be reserved for the number of bytes 
+   // you store in your metadata
+	pub const MetadataDepositPerByte: Balance = 1;
+
+   // Maximum lenght for the asset symbol and friendly name
+   pub const StringLimit: u32 = 50;
+}
+```
+
+It is important to note that every built-in module has a different purpose, and, therefore, each of them has different needs in terms of the parameters that must be configured. The following code snippet implements the trait and configures the module Assets, using types and the constants defined previously in the `parameter_types!` macro:
+
+```rust
+// Implementing the Assets config trait for the runtime
+impl pallet_assets::Config for Runtime {
+   
+   // Stores the balances in an unsigned integer of 128bits
+	type Balance = u128;
+   // The id of an asset can be defined as an unsigned integer of 64 bits
+	type AssetId = u64;
+   // Uses module Balances as mechanism for currency operations
+	type Currency = Balances;
+
+   // Configure the module by referencing the previously
+   // defined constants
+
+	type AssetDeposit = AssetDeposit;
+	type MetadataDepositBase = MetadataDepositBase;
+	type MetadataDepositPerByte = MetadataDepositPerByte;
+	type ApprovalDeposit = ApprovalDeposit;
+	type StringLimit = StringLimit;
+   
+   // More configuration
+   ...
+}
+```
+
+??? code "View the complete script"
+
+    ```rust
+    --8<-- 'code/basic-substrate/built-in-pallet-configuration.rs'
+    ```
+
+The complete configuration of the module contains more parameters, to view a detailed description of each of them, refer to the [official config trait for the Assets module documentation](https://paritytech.github.io/substrate/master/pallet_assets/pallet/trait.Config.html){target=_blank}.
+
+### Add the Module to the Runtime {: #add-module-to-runtime }
+
+In the same `lib.rs` file referenced in the previous section, there is a segment enclosed in the macro 'construct_runtime!()'. This is where the pallet must be added to be included in the runtime. Since the example is based on the EVM template, the runtime is already configured to include many modules, including the modules for system support, the modules to add the Ethereum compatibility layer, the modules to support the Tanssi protocol, balances, and now also Assets:
```

Comment:
```suggestion
In the same `lib.rs` file referenced in the previous section, there is a segment enclosed in the macro `construct_runtime!()`. This is where the pallet must be added to be included in the runtime. Since the example is based on the EVM template, the runtime is already configured to include many modules, including the modules for system support, the modules to add the Ethereum compatibility layer, the modules to support the Tanssi protocol, balances, and now also Assets:
```